modules/lumble/client/init.lua

```diff
@@ -977,6 +978,14 @@ function client:requestUserList()
 	self:sendTCP(msg)
 end
 
+function client:requestACL(channel_id)
```

Comment:
It may also be worthwhile to make this a channel metatable method in lumble/client/channel.lua

Something like...

```lua
function channel:requestACL()
	local query = packet.new("ACL")
	query:set("channel_id", self.channel_id)
	query:set("query", true)
	self:sendTCP(query)
end
```